code/game/antagonist/station/traitor.dm

```diff
@@ -175,6 +187,18 @@ var/datum/antagonist/traitor/traitors
 		to_chat(traitor_mob, "A portable object teleportation relay has been installed in your [R.name] [loc]. Simply enter the code \"[pda_pass]\" into the ringtone select to unlock its hidden features.")
 		traitor_mob.mind.store_memory("<B>Uplink Passcode:</B> [pda_pass] ([R.name] [loc]).")
 
+	else if(istype(R, /obj/item/modular_computer))
```

Comment:
This should require the user to select a modular device (laptop / tablet / wristbound) in the loadout and spawn with it.